README.md

```diff
@@ -19,6 +20,10 @@ deployment scheme.
 technology!](https://drive.google.com/file/d/0B667S2caJiy7QVpzVVFNQVdyaVE/view?usp=sharing) <<<**
 
 
+
+
```

Comment:
please remove the changes to the README.md file. your patch should only touch the new kubernetes.md file, i guess.